zerver/webhooks/bitbucket2/view.py

```diff
@@ -500,14 +510,15 @@ def get_user_info(dct: WildValue) -> str:
     # are technically still sending a message.
     log_unsupported_webhook_event(
         summary="Could not find display_name/nickname field",
+        request=request,
     )
 
     return "Unknown user"
 
 
-def get_actor_info(payload: WildValue) -> str:
+def get_actor_info(request: HttpRequest, payload: WildValue) -> str:
     actor = payload["actor"]
-    return get_user_info(actor)
+    return get_user_info(actor, request)
```

Comment:
The position of `request` is a bit inconsistent here -- is there a reason to not make `get_user_info` take `request` as its first parameter, like many of the other functions it's being added to?

Reply:
Yeah, I'd prefer for it to be the first parameter for just about any function accepting one.

Reply:
Okay, reordered.